README.md

```diff
@@ -4,6 +4,14 @@ SDK for building integrations in Hyperproof.
 
 ## Release Notes
 
+### 1.2.0
+
+- Update TypeScript to version 5.5.4
+- Update node-fetch to version 2.7.0
+- Update Express to version 4.21.0
+- Update Superagent to version 10.1.0
+- Update other dependencies to latest versions
```

Comment:
There are other changes here beyond the package upgrades.  At a minimum we should add "Fix various bugs" to the list.